internal/generator/services.go

```diff
@@ -89,6 +89,9 @@ func composeToServiceValues(
 	if composeServiceValues.Labels != nil {
 		lagoonType = lagoon.CheckServiceLagoonLabel(composeServiceValues.Labels, "lagoon.type")
 	}
+	if lagoonType == "" {
+		return ServiceValues{}, fmt.Errorf("No Lagoon type has been set for service %s. If a Lagoon type is not required for this service, please set the Lagoon type to 'none'", composeService)
```

Comment:
I would change all the `Lagoon type` to be `lagoon.type` so that it matches the label name that should be used. and then putting a small note about reading the documentation for supported types. 

Something like this:
```
No lagoon.type has been set for service %s. If this service is not meant to be deployed in Lagoon, please set the lagoon.type to 'none'. See the Lagoon documentation for supported service types.
```

Reply:
Just saw Toby also commented on changes for this. So some sort of combo of the two would probably be good

Reply:
Updated